packages/bcp/src/cosmwasmconnection.ts

```diff
@@ -372,9 +372,12 @@ export class CosmWasmConnection implements BlockchainConnection {
 
     // tslint:disable-next-line: deprecation
     const accountForHeight = await this.restClient.authAccounts(senderAddress, response.height);
```

Comment:
What is with the deprecation linter issue?

Reply:
The field `this.restClient` is marked as depreated in order to encourace using the CosmWasmClient, which is missing a few features right now.